cli/ioctl/README.md

```diff
@@ -5,10 +5,11 @@
     
     Available Commands:
       account     Deal with accounts of IoTeX blockchain
+      block       Deal with block of IoTex blockchain
```

Comment:
let's call it blockchain or bc, "block" means something different. also rename the file/folder pls